lib/modules/datasource/orb/index.ts

```diff
@@ -27,9 +27,10 @@ export class OrbDatasource extends Datasource {
     super(OrbDatasource.id);
   }
 
-  override readonly customRegistrySupport = false;
+  override readonly customRegistrySupport = true;
 
   override readonly defaultRegistryUrls = ['https://circleci.com/'];
+  override readonly registryStrategy = 'merge';
```

Comment:
is it how circle CI works? can a single orb exist on multiple registries?

Reply:
I wasn't quite sure what to pick here. Looks like hunt is a better option based off of my reading.